libcontainer/container_linux.go

```diff
@@ -581,6 +584,8 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 	defer criuServer.Close()
 
 	args := []string{"swrk", "3"}
+	logrus.Debugf("Using CRIU %d at: %s", c.criuVersion, c.criuPath)
+	logrus.Debugf("Using CRIU with following args: %s", args)
```

Comment:
Do you know of a better way to format this because with a lot of verbose parameters it's kinda hard to read the output?